mlflow/pyfunc/__init__.py

```diff
@@ -460,7 +460,7 @@ def predict(*args):
 
 
 def save_model(path, loader_module=None, data_path=None, code_path=None, conda_env=None,
-               model=Model(), python_model=None, artifacts=None):
+               mlflow_model=Model(), python_model=None, artifacts=None):
```

Comment:
Thanks for catching the inconsistency here with implementations in other MLflow flavors, such as `mlflow.pytorch.save_model`. Unfortunately, making this kwarg chance would create a backwards incompatibility issue for any users who currently specify `mlflow_model`. We could make this change and add a kwargs wildcard at the end that checks for the presence of `mlflow_model` to achieve backwards compatibility. However, for the scope of this work, I think it's best if we remove these changes and follow up later.

Reply:
The reason I made these changes were that mlflow.Model.log assumes that the mlflow_model parameter exists. As such, we would need to include mlflow_model in the kwargs at the minimum. 

In terms of the backwards compatibility, I agree - the reason I made the change was that I don't see `model` supported as a parameter in the docs [here](https://www.mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#mlflow.pyfunc.save_model)

Reply:
Alternatively, see: https://github.com/mlflow/mlflow/pull/1514/commits/04bb467b57ca352f53eacb87afc6ba6a1d209777 for an alternative solution which was my first try at this.